bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-dropdown/component.jsx

```diff
@@ -273,7 +283,12 @@ class UserDropdown extends PureComponent {
       actions.push(this.makeDropdownItem(
         'setstatus',
         intl.formatMessage(messages.statusTriggerLabel),
-        () => this.setState({ showNestedOptions: true, isActionsOpen: true }),
+        () => this.setState(
+          {
+            showNestedOptions: true,
+            isActionsOpen: true,
+          }, Session.set('dropdownOpen', true),
+        ),
```

Comment:
Does Settings dropdown need to call ` Session.set('dropdownOpen', true)`?

Reply:
It shouldn't need to. The user dropdown does in order to handle the nested emoji menu.